core/domain/feedback_domain.py

```diff
@@ -50,7 +80,7 @@ def to_dict(self):
             'summary': self.summary,
             'thread_id': self.get_thread_id()
         }
-
+# TODO: add docstring here
```

Comment:
My bad, I forgot to hit save in the editor before submitting the code. I'll fix this now.